apps/api/src/common/config/index.ts

```diff
@@ -12,7 +12,10 @@ dotenv.config();
 
 const logger = Logger.getInstance();
 
-const env = validationSchema.safeParse(process.env);
+const env = validationSchema.safeParse({
+    ...process.env,
+    L2_RPC_URLS_MAP: JSON.parse(process.env.L2_RPC_URLS_MAP || "{}"),
```

Comment:
A minor drawback this `JSON.parse` here could have is that it might throw when parsing an invalid JSON string, causing the nice summary of validation errors to not be generated.

I was curious to see if we could leverage `zod` for this; found [this conversation](https://github.com/colinhacks/zod/discussions/2215#discussioncomment-5356276) that might be useful if you want to move that `JSON.parse` to a schema.

Reply:
nice catch here

Reply:
nice one here, i used the suggestion in the discussion 🙌